src/bulk/common.ts

```diff
@@ -1268,6 +1264,18 @@ export abstract class BulkOperationBase {
     batchType: BatchType,
     document: Document | UpdateStatement | DeleteStatement
   ): this;
+
+  private shouldForceServerObjectId(): boolean {
+    if (typeof this.s.options.forceServerObjectId === 'boolean') {
+      return this.s.options.forceServerObjectId;
+    }
+
+    if (typeof this.s.collection.s.db.options?.forceServerObjectId === 'boolean') {
+      return this.s.collection.s.db.options?.forceServerObjectId;
```

Comment:
```suggestion
      return this.s.collection.s.db.options.forceServerObjectId;
```

Not in scope but db.options is never nullish according to the db constructor, any concerns just cleaning that up?

Reply:
I chose to leave the `?` chaining.